src/connector/src/kafka/mod.rs

```diff
@@ -23,7 +23,7 @@ pub use split::*;
 
 const KAFKA_SYNC_CALL_TIMEOUT: Duration = Duration::from_secs(1);
 
-const KAFKA_CONFIG_BROKER_KEY: &str = "kafka.broker";
+const KAFKA_CONFIG_BROKERS_KEY: &str = "kafka.brokers";
```

Comment:
according to docs, we may use `kafka.bootstrap.servers` here

Reply:
Yes, I will eventually fix the document.